Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -28,7 +28,12 @@ def edit
       EOS
     end
 
-    paths = args.named.to_formulae_paths.presence
+    paths = args.named.to_formulae_paths.select do |path|
+      formula_name = File.basename(path.basename, ".*")
+      Formulary.factory(formula_name)
+      rescue FormulaUnavailableError => e
+        raise UsageError, "#{e.message}. Run #{Formatter.identifier("brew create $URL")} to create a new Formula!"
+    end
```

Comment:
```suggestion
    end.presence
```

otherwise the`paths ||=` below will never trigger.

Reply:
thank you for the suggestion, I'll fix this.